tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -135,6 +134,11 @@ object Symbols {
         assert(false, s"cannot access owner, ${this.name} is local or not declared within any scope")
     }
 
+    /** What is the closest enclosing package of this symbol? Returns this if this is a package */
+    private[tastyquery] final def closestPackage: PackageSymbol = this match
+      case pkg: PackageSymbol => pkg
+      case sym: Symbol        => sym.owner.nn.closestPackage // owner of regular symbols is never null
```

Comment:
```suggestion
      case pkg: PackageSymbol => pkg
      case sym: TermOrTypeSymbol => sym.owner.closestPackage
```
`Symbol` is partitioned into `PackageSymbol` and `TermOrTypeSymbol`, and the latter's `owner` is known to be non-null. :)

Reply:
thanks